query/src/lib.rs

```diff
@@ -43,14 +43,32 @@ pub type SrcVarName = String;          // Do not include the required syntactic
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct Variable(pub PlainSymbol);
 
-#[derive(Clone,Debug,Eq,PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
It doesn't, and for no good reason I have a preference for the no-spaces style, but Jim's book uses spaces.